### PR TITLE
0.9.5 release

### PR DESCRIPTION
**Pull request by cam-parra**
Signed-off-by: Cam Parra <caeparra@gmail.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,87 +1,120 @@
...
+        git clone https://github.com/hyperledger/indy-sdk.git
+        cd indy-sdk
+        git checkout stable 
+        cd libindy
+        cargo clean 
+        cargo update
+        cargo build
+    
+    Libsovtoken build needs to know how to find Indy-SDK. This is done
+    through the environment variable LIBINDY<sub>DIR</sub>.
```
Comment (brentzundel) [inline README.md]: Do you intend this to be LIBINDY<sub>DIR</sub> and not LIBINDY_DIR?

File: README.md [hunk 2]
```diff
@@ -1,87 +1,120 @@
...
-If you update libsovtoken code from github you need to re-build libsovtoken as follows:
-1) cargo clean
-2) cargo update
-3) cargo build
+## Compiling libsovtoken and running tests
 
-The cargo update command is necessary to make sure you get latest from the cargo crates in github
+1.  Make sure you meet the requirements above, including building the
+    projects, if you elect to use source code.
+2.  Get latest for libsovtoken from master at
```
Comment (brentzundel) [inline README.md]: I think you shouldn't specify which branch

File: README.md [hunk 3]
```diff
@@ -1,87 +1,120 @@
...
+3.  The source code is in a sub-directory called libsovtoken
+4.  Run the following commands from the libsovtoken subdirectory
+    1.  cargo update
+    2.  cargo build
 
-## How To Contribute
+5.  Run the tests to ensure everything is in good order.
+    1.  cargo test
+
+***Please keep in mind*** (at the time this document was written) some of
```
Comment (brentzundel) [inline README.md]: Is this still true?

File: README.md (hunk 1 above)
Comment (cam-parra) [reply README.md]: This was an exporter error. I will fix it.

**Comment by cam-parra** [comment]
@brentzundel please review the new description of libsovtoken

File: README.md [hunk 4]
```diff
@@ -1,87 +1,113 @@
...
-If you update libsovtoken code from github you need to re-build libsovtoken as follows:
-1) cargo clean
-2) cargo update
-3) cargo build
-
-The cargo update command is necessary to make sure you get latest from the cargo crates in github
+
+1.  Make sure you meet the requirements above, including building the
+    projects, if you elect to use source code.
+2.  Get latest for libsovtoken at
```
Comment (brentzundel) [inline README.md]: Should probably update this URL.

**Pull request merged**